chrome/sunset.js

```diff
@@ -104,26 +104,49 @@ Sunset.shouldShowNotification_ = function(start, now, index) {
  * @private
  */
 Sunset.maybeShowNotification_ = function() {
-  chrome.storage.sync.get(null, function(data) {
-    // Populate the starting date with today's date if it isn't already set.
-    var starting_date = Date.parse(data.start) ? new Date(data.start) : new Date();
-    if (data.start != starting_date.toString())
-      chrome.storage.sync.set({ "start": starting_date.toString() });
-
-    // Read the index of the notification to be shown.
-    var index = parseInt(data.index, 10);
-
-    // Reset the index if the stored data are invalid or not present.
-    if (isNaN(index) || index < 0) {
-      index = 0;
-    }
-
-    // Trigger notification if we have reached the date suggested
-    // by the timeline. This check is not required for the first notification.
-    var now = new Date();
-
-    if (Sunset.shouldShowNotification_(starting_date, now, index))
-      Sunset.showNotification_(index);
+  chrome.storage.local.get(null, function(local_data) {
```

Comment:
What happens if the user isn't signed in? The docs are iffy here: "Even if a user disables syncing, storage.sync will still work. In this case, it will behave identically to storage.local." Does that mean they both store data locally, or that they look at _the same_ data?